redis/src/client.rs

```diff
@@ -208,6 +208,7 @@ impl Client {
     /// [multiplexed-connection]: aio/struct.MultiplexedConnection.html
     #[cfg(feature = "connection-manager")]
     #[cfg_attr(docsrs, doc(cfg(feature = "connection-manager")))]
+    #[deprecated(note = "use get_connection_manager_with_backoff instead")]
```

Comment:
Can we continue to have a non-`with_backoff` method that supplies the defaults (i.e., that just removes `tokio` from the name of the current method)?

Reply:
done

Reply:
Awesome, thanks!